docs/INSTALL.rst

```diff
@@ -16,8 +16,7 @@ available:
 - A supported version of Python, including the development support if
   installed from system-level packages.  Supported versions include:
 
-  * 2.7
-  * 3.5 - 3.8
+  * 3.5 up to 3.8
```

Comment:
Don't we use "Python 3.5 and higher" in other places instead of limiting to 3.8?

Reply:
I explicitly changed it some weeks ago to mention the currently supported maximum version of Python as it tells about the actual current status of supported versions. (It is more explicit than the implicit meaning that only Python 3 versions are supported which are known at the time when this part of the documentation was written.)

Yes, this requires to adapt the documentation once we support Python 3.9, but this is also an explicit step.